specs/multiera/lib.cddl

```diff
@@ -1,8 +1,11 @@
 ; babbage block
 block = _CDDL_CODEGEN_EXTERN_TYPE_
 
-mutli_era_block =
-    shelley_block ; @name Shelley
+byron_block = _CDDL_CODEGEN_EXTERN_TYPE_
+
+multi_era_block =
+    byron_block   ; @name byron
```

Comment:
`@name Byron`?

Reply:
It names the rust enum variant to that instead of deriving it from the name. MultiEraBlockByron(ByronBlock)

Reply:
I mean Byron should be capitalized:

```
  byron_block ; @name byron -> @name Byron
   / shelley_block ; @name Shelley
   / allegra_block ; @name Allegra
   / mary_block    ; @name Mary
   / alonzo_block  ; @name Alonzo
```

Reply:
I guess but it gets converted to UpperCamelCase anyway so it doesn't change the output code.

Reply:
yes, that's for naming consistency